homeassistant/components/alpha_vantage/sensor.py

```diff
@@ -79,9 +77,13 @@ def setup_platform(
 ) -> None:
     """Set up the Alpha Vantage sensor."""
     api_key: str = config[CONF_API_KEY]
+    scan_interval: str = config[CONF_SCAN_INTERVAL]
     symbols: list[dict[str, str]] = config.get(CONF_SYMBOLS, [])
     conversions: list[dict[str, str]] = config.get(CONF_FOREIGN_EXCHANGE, [])
 
+    _LOGGER.debug("Scan interval = %s", scan_interval)
+    self.SCAN_INTERVAL = timedelta(minutes=scan_interval)
```

Comment:
This doesn't work. The scan interval can already be changed in the platform schema as pointed out in the discussion above.